ONSdigital/dis-wagtail-alpha
# Major design system version update

### What is the context of this PR?
There has been a major update to the design system, which is now using a base 16 grid. Although this means some rework on the Nov prototype work so far, it will be better to switch in the long run.

It includes an update to the `isoCode` variable name in the header template, which requires some code in our build to be updated to match.

### How to review

- Check out this branch, then run `fab build`
- Once you have started and entered the docker container, you may need to run `make load-design-system-templates`
- Go to the home page of the site, and inspect an element with class `ons-container`. If you go to view the *computed* value of the left and right padding, it should be 16px (not 18px).

![Screenshot 2024-10-11 at 10 05 38](https://github.com/user-attachments/assets/73f747b9-eb21-453a-b94e-bf374b48b181)
